app/addons/components/tests/stringEditModalSpec.react.jsx

```diff
@@ -70,7 +72,7 @@ define([
           container
         );
         var modal = TestUtils.findRenderedComponentWithType(el, Modal);
-        var modalEl = React.findDOMNode(modal.refs.modal);
+        var modalEl = ReactDOM.findDOMNode(modal.refs.modal);
 
         TestUtils.Simulate.click($(modalEl).find('#string-edit-save-btn')[0]);
```

Comment:
Thanks Robert, yeah I'm going to use this technique instead. It's just a test: doesn't have to be beautiful, as long as it's tested!